Add docs for Python .ljust() string method (#3339)
Codecademy/docs: content/python/concepts/strings/terms/ljust/ljust.md

---
Title: '.ljust()'

> Description: 'Left align the string, using a specified character (space is default) as the fill character'

Reviewer: ```suggestion
Description: 'Justifies a given string to the left using the specified padding.'
```

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Strings'
  - 'Methods'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> The `ljust()` method will left align the string, using a specified character (space is default) as the fill character.

Reviewer: ```suggestion
The **`ljust()`** method will justify a given string to the left, using a specified character (space is the default) as the fill character.
```

## Syntax

The `.ljust()` method is called on a string:

```pseudo
string.ljust(length, character)
```

> - `length`: ***Required***. The length of the returned string

Reviewer: ```suggestion
- `length`: (Required) The length of the returned string
```

> - `character`: ***Optional***. A character to fill the missing space (to the right of the string). Default is " " (space).

Reviewer: ```suggestion
- `character`: (Optional) A character to use as padding instead of whitespace (to the right of the string).
```

## Example

> Using the letter "O" as the padding character:

Reviewer: ```suggestion
The example below uses the letter `"O"` as the padding character in the `ljust()` method:
```

```python
txt = "banana"

x = txt.ljust(20, "O")

print(x)

> #Output: bananaOOOOOOOOOOOOOO

Reviewer: put the output in a ```shell block
